mobile/src/components/course/components/my-mentor/components/my-mentor-card/my-mentor-card.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC, ReactElement } from 'react';
+
+import defaultCourseImage from '~/assets/images/avatar-default.png';
+import { ButtonVariant } from '~/common/enums/enums';
+import { UsersGetResponseDto } from '~/common/types/types';
+import { Button, Image, Text, View } from '~/components/common/common';
+import { getImageUri } from '~/helpers/helpers';
+
+import { styles } from './styles';
+
+type Props = {
+  mentor: UsersGetResponseDto | null;
+  handleMentorCardShown: () => void;
```

Comment:
handlers in props should be named via on
```suggestion
  onChangeMentor: () => void;
```